### ISSUE TITLE
Reduce amount of compiled output

### ISSUE THREAD

**Issue by HMIDeer** [DominoKit/domino-ui#254]
I've just checked the compiled output and its folders and there is a 3,52 MB sized folder called plugins which contains such as stepper, chosen , ...Within each of the sub folders several font formats are located. I am not using a stepper but its still there. The same with the themes in a folder called css/themes all *.css files for all themes are contained but i am using only the blue one and no theme chooser for the user

Additionally the global defined fonts are also present in several formats, there isn't any possibility to choose one format.

**Comment by HMIDeer** [comment]
My suggestion for the fonts would be to add them depended of a flag where the user can define its needed browser support e.g. remove *.eot if IE < 6 should't be supported

**Comment by TDesjardins** [comment]
It's not just the compiled output... it is also the traffic, the requests and the parsing of the not needed resources. This is the result of a lighthouse check of Domino UI demo (performance: 69):

![image](https://user-images.githubusercontent.com/1841306/64908256-ea733700-d6fd-11e9-9ce0-ca350af9da58.png)

Seems that there are lot things to improve. Here are some suggestions:

* compress all resources (css, external script, fonts...)
* add ability to choose a color/theme (I would also welcome to load no color/theme)
* only add resources for components when they are used

**Comment by niloc132** [comment]
Compression of static resources is really up to the webserver - if you're loading this from dominokit.github.io, the compression is at their mercy, though it might be possible to at least provide a gzipped copy and their server will provide the compressed copy instead? Might be worth a try, though I think the Precompress linker will do that automatically for you just for your .cache.js at least (again, assuming the github.io server, like jetty, actually knows how to serve compressed content).

**Comment by TDesjardins** [comment]
> Compression of static resources is really up to the webserver - if you're loading this from dominokit.github.io, the compression is at their mercy, though it might be possible to at least provide a gzipped copy and their server will provide the compressed copy instead? 

Yep, dominokit.github.io already serves the files gziped. The lighthouse check shows already the compressed sizes. If you are loading it from a server without gzip the css stuff costs 1 MB more traffic (performance index is then on my machine between 10 and 20). So if the resources are not precompressed the gzip is a must have. The .cache.js files are not that problem. CSS files like colors.css (410 KB) play a role.

**Comment by vegegoku** [comment]
@TDesjardins @niloc132 we are already taking steps to fix some of these issues, but we also need this  issue #161 an then we can work more freely on this one.
